.gitignore

```diff
@@ -30,3 +30,6 @@ changelog.txt
 /patches_old_bak/
 *.py
 *.7z
+
+# OS specific files
+.DS_STORE
```

Comment:
Something like this belongs in your local, global `.gitignore`.

Reply:
Should I just remove the commit then?

Reply:
Or revert this change.